resotolib/resotolib/baseplugin.py

```diff
@@ -214,6 +214,11 @@ def collect(self) -> None:
         """Collects all the Cloud Resources"""
         pass
 
+    @staticmethod
+    def auto_enabled() -> bool:
```

Comment:
It feels like we could use a different name. What about `ready_to_start`?

Reply:
Idk, the method determines whether the collector will be automatically enabled. It doesn't say anything about whether the collector is ready it also doesn't configure the collector. All it does is automatically add the collector name to the list of enabled collectors. Hence the name auto_enabled(). Of the suggestions auto_enableable would fit, but enableable is also not a common word you'd find in most dictionaries. What about `auto_ activatable`?